bcgov/mds
# Hot fixes - MineSpace project Descriptions

# Main

- Added validation to the phone_ext on project description form
- update the document types accepted

# Other

- N/A

# How to test

- Ensure that you can upload specific file types on the MineSpace document upload.

# Notes

- Any notes/screenshots for this PR?
- N/A
